packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -42,27 +49,27 @@ async function fetchPairData(
   );
 }
 
-//Adjustment of the difference in coin decimals, for example WETH has 18 decimals, USDT and USDC have 6.
-function adjustRate(rate: Price<Token, Token>): Fraction {
+// Returns a rate in the biggest units of destination token not the smallest units.
```

Comment:
I think it would be great to add a rate conversion example here in the code comment and explain why biggest units